docs/rewards/rewards-troubleshooting.mdx

```diff
@@ -16,7 +16,7 @@ It seems that you haven't connected a wallet to your account yet. Please note th
 1. Learn [how to add or edit a wallet address](/wallet/add-edit-wallet-address).
 2. Then, follow the instructions for [joining Arbitrum Sepolia and see your $WXM tokens](/wallet/how-to-join-arbitrum-and-see-your-tokens#add-arbitrum-sepolia-network-to-metamask).
 
-Once you've completed the above steps, you'll be all set to claim your rewards. Check back tomorrow and see "[How to Claim Rewards](/project/rewards-claiming)" for further guidance.
+Once you've completed the above steps, you'll be all set to claim your rewards. Please wait until the [Quality of Data (QoD) algorithm](/project/quality-of-data) completes. This usually takes until the following day and see "[How to Claim Rewards](/project/rewards-claiming)" for further guidance.
```

Comment:
This is wrong, it's not the QoD algorithm that produces the "NO_WALLET" error. The user needs to wait for the next run of the reward mechanism.

Reply:
fixed